Improve helptext top-level description (#158)
con/duct: README.md

            [-t {all,system-summary,processes-samples}]
            command [command_args ...] ...

duct is a lightweight wrapper that collects execution data for an arbitrary
command.  Execution data includes execution time, system information, and
resource usage statistics of the command and all its child processes. It is
intended to simplify the problem of recording the resources necessary to

> execute a command to improve reproducibility, particularly in an HPC
> environment.

Reviewer: ```suggestion
execute a command, particularly in an HPC environment.
```

since AFAIK there is little to nothing about reproducibility aspect here.

> Resource usage is determined by polling, (of frequency sample_interval).

Reviewer: odd syntax + frequency is a reciprocal to interval

```suggestion
Resource usage is determined by polling (at a sample-interval).
```

note change of `_` to `-` for consistency with other similar use below

> During execution, duct produces a Newline Deliminated JSON file with one data

Reviewer: ```suggestion
During execution, duct produces a JSON lines (see https://jsonlines.org) file with one data
```

since it is not JSON line per se

> point recorded for each report, (of frequency report-interval).

Reviewer: see above

```suggestion
point recorded for each report (at a report-interval).
```

            [-t {all,system-summary,processes-samples}]
            command [command_args ...] ...

duct is a lightweight wrapper that collects execution data for an arbitrary
command. Execution data includes execution time, system information, and

> resource usage statistics of the command and all its child processes.

Reviewer: should mention that it samples that process and children periodically, so that information could be used to later analyze/plot command's resources consumption.  And that it might be useful to declare needed resources for allocation on HPC tasks, or for troubleshooting.  Also note that it stores all the standard output and error into external files.
Do we have somewhere listing of env vars to configure (may be in README for now)? should refer to that description here too IMHO.

Author: > should mention that it samples that process and children periodically, so that information could be used to later analyze/plot command's resources consumption. And that it might be useful to declare needed resources for allocation on HPC tasks, or for troubleshooting. Also note that it stores all the standard output and error into external files.

+1

> Do we have somewhere listing of env vars to configure (may be in README for now)? should refer to that description here too IMHO.

Its not listed anywhere atm, +1 to add here.
